### DISCUSSION TITLE
Door message on 2.0.15 on OSX

### DISCUSSION THREAD

**Discussion by dkag123** [winder/Universal-G-Code-Sender#2234] [Help]
Anyone know what this 'door' message is?
I keep getting it now and can no longer work because I have no clue what it is or what is causing it.
I never seen this message before.
thanks

**Comment by breiler** [comment]
Read about the door feature in GRBL here:
https://github.com/gnea/grbl/wiki/Grbl-v1.1-Interface#feedback-messages

**Reply by dkag123** [reply]
thanks.
Unfortunately I still have no clue what its talking about.
I just started getting this message out of no where, tried changing USB ports but no luck.
I didn't make any changes to the system (OSX Mojave 10.14.6) or the CNC machine 
I may have another motherboard to hook up if all else fails...and try that
Thanks again for your help

**Reply by breiler** [reply]
The DOOR-state can be triggered from a special door command (in UGS there is a button for it) or from an input pin on your controller. 

Without knowing what controller/firmware you are using it is difficult to help. But I find it more likely that it is caused by the input pin being triggered somehow. Is it left floating/unconnected, then might help pulling it to GND.

**Reply by dkag123** [reply]
Yes, I tried the door button (when it's highlighted) but it isnt having any effect. Mostly that button is greyed out when it happens.
I turn the machine on, everything is fine and working, then about a minute or two later I lose all jog control, a minute or so after that the door message appears. I have the SainSmart Genmtisu GRBL Controller Board for CNC Router Machine 3018-PROVer/ 3018-PROVer Mach3/ 3018-MX3   https://www.amazon.com/gp/product/B0834THBLC/ref=ppx_yo_dt_b_asin_title_o03_s00?ie=UTF8&psc=1

**Reply by breiler** [reply]
Their product page doesn't link to any manual or firmware source so I can only guess... 
https://www.sainsmart.com/collections/genmitsu-cnc-replacement-upgrade-parts/products/genmtisu-grbl-controller-board-for-3018-prover-3018-mx3?utm_source=homepage&utm_campaign=new_arrival

Do you have the e-stop connected?

**Reply by dkag123** [reply]
yes, if I disconnect it, it wont connect at all to the machine.. I have always had it connected but with it locked in the pressed state, To me it seems backwards but it has always worked this way so I let it be.  Not sure what else I can try but to change the motherboard.. I have another one (same make and model)
